docs/docs/configuration/overview.md

```diff
@@ -28,3 +28,26 @@ export default {
   }
 };
 ```
+
+## Adding Rollup Plugins
+
+Whenever you want to add special handling of files/requests you will need to add ut to both the dev server and the rollup production build.
```

Comment:
```suggestion
For some projects you might want to enable non-standard behaviours like importing JSON files as JavaScript. 

```js
import data from './data.json';
```

You can accomplish this with rollup and dev server plugins. Make sure to add both the dev-server plugin as well as the rollup plugin, so that the behaviour is the same during development as it is in the production build.
```